includes/dashboard/FurtherUserOptions.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/* 
+ * this is to view users category wise
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+if ( !(isset($_POST['tab']) && isset($_POST['levelId'])) )
+{
+	echo "Sorry, there is some problem<br>";
+	die ();
+}
+
+
+
+$tab = $_POST['tab'];
+$levId = $_POST['levelId'];
+
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $catDB );
+
+if( $sqlConn->connect_errno ) 
+{
+	echo "Error connecting database. Please check your database credentials and update.";
+	die();
+}
+
+$newLevId = $levId."_[0-9]{1,}";
+$qry = "SELECT * FROM $tab WHERE $levels REGEXP \"^$newLevId$\"";
+$stmt = $sqlConn->prepare($qry);
+
+if ( ! $stmt->execute() )
+{
+	echo"there is a problem with database<br>";
+	die ();
+}
+
+$result = $stmt->get_result ();
+
+$catType = str_replace ( "L", "C", $levId );
```

Comment:
__[code_sniffer]__
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->